superset/assets/visualizations/time_table.jsx

```diff
@@ -70,10 +70,10 @@ function viz(slice, payload) {
           // Period ratio sparkline
           sparkData = [];
           for (let i = c.timeRatio; i < data.length; i++) {
-            sparkData.push(data[i][metric] / data[i - c.timeRatio][metric]);
+            sparkData.push(d3.round(data[i][metric] / data[i - c.timeRatio][metric], 2));
```

Comment:
I feel like we shouldn't round here, instead change the number formatting when assembling the tooltip